### ISSUE TITLE
Change link colors from orange to something else

### ISSUE THREAD

**Issue by nabeelio** [nabeelio/phpvms#826]
Accessibility, blue

![image](https://user-images.githubusercontent.com/99736/93269013-324af500-f77c-11ea-88c9-3d39f9cbdc83.png)

**Comment by Botric** [comment]
Hyperlinks get the colour from resources/sass/now-ui/now-ui-kit/_variables.scss
 `<49> $primary-color:              #f96332 !default;`
if this helps

**Comment by nabeelio** [comment]
Yup, I know, just haven't gotten to it yet. Gotta look at a color wheel

**Comment by arthurpar06** [comment]
I looked a bit into the kind of blue that would comply with WCAG AAA, especially this one `#0057AD`. It's ok with WCAG AAA but when I try it, I find that it stands out less than the orange despite meeting the standard.

https://webaim.org/resources/contrastchecker/?fcolor=0057AD&bcolor=FFFFFF

With actual orange links:
![image](https://github.com/nabeelio/phpvms/assets/41431456/43f195c7-1c4b-4389-ab7b-e7b9fcafddaa)

With `#0057AD` blue links:
![image](https://github.com/nabeelio/phpvms/assets/41431456/2ece813d-5455-46bb-84c9-b075894e20ff)
![image](https://github.com/nabeelio/phpvms/assets/41431456/7ea070ad-96c9-4106-9319-cef9f9ad63cc)
